pandas/__init__.py

```diff
@@ -346,3 +346,8 @@
     "unique",
     "wide_to_long",
 ]
+
+from pandas.core.accessor import DataFrameAccessorLoader
+
+DataFrameAccessorLoader.load()
```

Comment:
Why a class with a single method?

And why only doing this for `DataFrame`, not for `Series`?

Reply:
> Would you recommend covering that already in this PR?

Yes, I'd say so. It won't be much more complicated and any libraries that provide both Series and DataFrame accessors wanting to will want these at the same time.